package.json

```
@@ -13,12 +13,13 @@
     "husky": "^4.2.3"
   },
   "scripts": {
-    "lint": "yarn workspaces run lint"
+    "lint": "yarn workspaces run lint",
+    "heroku-postbuild": "yarn workspaces run heroku-postbuild"
```

If the command is only supposed to run in a single project, then it's best not to use `yarn workspaces`, this could be used instead:

```suggestion
    "heroku-postbuild": "yarn workspace api heroku-postbuild"
```

You can then get rid of the no-op `echo` commands in the other projects.